extensions/extensions-helm/aissemble-configuration-store-chart/README.md

```diff
@@ -0,0 +1,42 @@
+| deployment.serviceAccountName           | The service account for the pod                                                                                                  | No                | Default user in the cluster namespace                                                                                                                                          | 
+| deployment.automountServiceAccountToken | Whether or not to mount service account token. Should be disabled unless the pod requires specific access to the Kubernetes APIs | No                | false                                                                                                                                                                          | 
+| deployment.env                          | The environment variables for the pod. Used to define the base and environment uri                                               | No                | None                                                                                                                                                                           | 
+| deployment.args                         | The args for the pod                                                                                                             | No                | None                                                                                                                                                                           | 
+| deployment.restartPolicy                | The restart policy                                                                                                               | No                | Always                                                                                                                                                                         |
+| deployment.ports                        | The deployment ports                                                                                                             | No                | - name: http-1 <br/>&emsp;&emsp;containerPort: 8080 <br/>&emsp;&emsp;protocol: TCP                                                                                             |
+| service.spec.type                       | The service type                                                                                                                 | No                | None                                                                                                                                                                           | 
+| service.spec.ports                      | The service ports                                                                                                                | No                | - name: http <br/>&emsp;&emsp;port: 8083<br/>&emsp;&emsp;protocol: TCP<br/>&emsp;&emsp;targetPort: 8080                                                                        |
```

Comment:
Q/S: Can we simplify this so that we just use the same port for the deployment spec and the service spec? Could always add more specific overrides later if they're needed.  Also recommend dropping `spec` e.g. `service.ports`.

Reply:
Yeah thats not a problem.